packages/core/helper-plugin/package.json

```diff
@@ -32,7 +32,7 @@
     "build:production": "cross-env NODE_ENV=production webpack-cli",
     "create:index": "node ./createBuildIndex.js",
     "watch": "yarn create:index && cross-env NODE_ENV=development webpack-cli -w",
-    "test": "echo \"no tests yet\"",
+    "test:front": "cross-env IS_EE=true jest --config ./jest.config.front.js",
```

Comment:
I don't think we need the IS_EE env variable in this package, we don't plan on making components that are for the ee only.